wtm.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: page
+title: WTM
+permalink: /wtm
+---
+
+# Women Techmakers Pisa
+
+### Cos'è il WTM?
+
+**Women Techmakers** è un programma di Google rivolto alle donne che amano la tecnologia.  
+Noi del WTM di Pisa, affrontiamo questo tema rivolgendoci non solo ad un pubblico femminile perchè riteniamo sia importante valorizzare le persone indistintamente e non in base al proprio genere.  
```

Comment:
Penso che **non solo** vada in grassetto dato che e' molto facile da confondere con **solo**